source/includes/_fund_acceptance.md

```diff
@@ -617,6 +623,13 @@ print(data.decode("utf-8"))
 }
 ```
 
+### Request Headers
+
+Parameters | Type | Description
+---- | ---- | ----
+X-Api-Key | String | API Key for establishing connection to this particular endpoint
+X-Oy-Username | String | The registered partner username which access is enabled for payment link product
```

Comment:
nitpick: The registered partner's username with enabled access for payment link product